examples/yolo/README.md

```diff
@@ -1,29 +1,33 @@
-# Examples of YOLO [1]
+# Examples of YOLO [1, 2]
 
 ## Performance
 PASCAL VOC2007 Test
 
 | Model | Original | ChainerCV (weight conversion) |
 |:-:|:-:|:-:|
+| YOLOv2 | 75.8 % * | 75.8 % |
 | YOLOv3 | 80.2 % | 80.2 % |
 
 Scores are mean Average Precision (mAP) with PASCAL VOC2007 metric.
 
+\*: Although the original paper [1] reports 76.8 %, the original implementation and provided weights archived this score.
```

Comment:
How about

```
Although the original paper [1] reports 76.8 %,, the darknet implementation and the provided weights achieved the lower score.
```

Also, is this a known issue? We can provide a link to a page that discusses this issue.

Reply:
I didn't know this issue. After I find the issue (or make an issue by myself), I will add a link.

Reply:
This article reports 75.8% (and 75.9%).
https://github.com/AlexeyAB/darknet#how-to-calculate-map-on-pascalvoc-2007